.circleci/config.yml

```diff
@@ -516,6 +512,14 @@ jobs:
       CUDA_VERSION: "9"
     <<: *pytorch_linux_build_defaults
 
+  pytorch_linux_xenial_cuda9_cudnn7_py2_caffe2_test:
+    environment:
+      JOB_BASE_NAME: pytorch-linux-xenial-cuda9-cudnn7-py2-caffe2-test
+      PYTHON_VERSION: "2.7"
+      CUDA_VERSION: "9"
```

Comment:
We need to add `DOCKER_IMAGE: "308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py2:256"` I think